Assets/RandomizeWires.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomizeWires : MonoBehaviour

```

Comment:
Move this to being a method within the WirePuzzleManager class rather than being a class on its own